Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -549,8 +553,12 @@ def splunk_submit_event_hec(hec_token, baseurl, event, fields, host, index, sour
     return response
 
 
-def splunk_submit_event_hec_command():
-
+def splunk_submit_event_hec_command(proxy):
+    if not proxy:
```

Comment:
Is there a reason not to use `handle_proxy` here?

Reply:
Thank you for the review comment. We can use handle_proxy. I was not aware of this common method mainly because the original author has not used it too. If you look at splunk_edit_notable_event_command, it did not use it hence i did not.  Please let me know if I should make the change.